src/EmployerAan/SFA.DAS.EmployerAan/Configuration/CommitmentsV2ApiConfiguration.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.EmployerAan.Configuration;
+
+[ExcludeFromCodeCoverage]
+public class CommitmentsV2ApiConfiguration : InnerApiConfiguration { }
```

Comment:
Remove the unnecessary use of base class from shared project. This base class is written for shared clients which we are not using.